lesson10/index.js

```diff
@@ -0,0 +1,67 @@
+const ul = document.getElementById('js-target');
+
+function addLoading() {
+  const img = document.createElement('img');
+  img.id = 'loading-img';
+  img.src = 'loading-circle.gif';
+  img.alt = 'loading';
+  ul.appendChild(img);
+}
+
+function removeLoading() {
+  const loadingImg = document.getElementById('loading-img');
+  loadingImg.remove();
+}
+
+function getAttributes() {
+  return new Promise((resolve) => {
+    setTimeout(() => {
+      const attributes = [
+        {
+          to: "bookmark.html",
+          img: "1.png",
+          alt: "画像1",
+          text: "ブックマーク"
+        },
+        {
+          to: "message.html",
+          img: "2.png",
+          alt:"画像2",
+          text: "メッセージ"
+        }
+      ];
+      resolve(attributes);
+    }, 3000);
+  })
+}
+
+async function fetchData() {
+  addLoading();
+  try{
+    return await getAttributes();
+  } catch(e) {
+    console.error(e)
```

Comment:
IMO
Argument names that identify the error are good.
Because `e` is often used as an argument in the event listener.
```suggestion
  } catch(error) {
    console.error(error)
```

Reply:
I agree with Mai-san!

Reply:
@nakagawamai @haru-programming 
Thanks for the review.
Sure. Not clear...

Fixed.
[3b406d1](https://github.com/tom106gunma/JavaScript-assignment/pull/11/commits/3b406d1a3d47b648632857e54f228ba614915aa4)